src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -3571,56 +3566,6 @@ private void SetQuotas()
 
                 Hashtable quotas = transportOption.ConstructQuotasAsHashtable();
 
-                int idleTimeOut = 0;
-
-                if (idleTimeOut != 0 && quotas.ContainsKey(WSManConfigurationOption.AttribMaxIdleTimeout))
```

Comment:
@iSazonov Is unreachable code a bug?

Reply:
I think so, yes

Reply:
@xtqqczze Please create new issues for such cases. We could think more and conclude if thus a bug.

Reply:
Issue opened: #13248